### PR TITLE
DCS-1277 new page layout

### PR DESCRIPTION
**Pull request by GurnankCheema**
page now looks like this. Note for this record, there is no pnc number

screenprint

Link to prototype
https://hmpps-rap-prototype.herokuapp.com/v10/prisoners/matthew-lee/confirm-arrival

### CODE REVIEW

File: server/views/pages/checkTransfer.njk [hunk 1]
```diff
@@ -1,75 +1,97 @@
...
+                                classes: "govuk-!-margin-bottom-0",
+                                rows: [
+                                    {
+                                    key: {
+                                        text: "Name",
+                                        classes: "no-border-bottom govuk-!-padding-bottom-0"
+                                    },
+                                    value: {
+                                        text: data.firstName + ' ' + data.lastName,
+                                        classes: "no-border-bottom govuk-!-padding-bottom-0 data-qa-name"
```
Comment (GurnankCheema) [inline server/views/pages/checkTransfer.njk:29]: set 'data-qa-*' as a class solely for integration tests. Govuk summary list component doesn't have options to set data attributes for summary rows.

File: server/views/pages/checkTransfer.njk [hunk 2]
```diff
@@ -1,75 +1,97 @@
 {% extends "../partials/layout.njk" %}
-{% set pageTitle = "Check transfer" %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% set pageTitle = "Confirm prisoner's arrival" %}
```
Comment (GurnankCheema) [inline server/views/pages/checkTransfer.njk:3]: Not 100% sure whether page title should be 'Confirm prisoner's arrival' or 'This person is being transferred from another establishment' 
Former was shorter so used that.

**Review by andrewrlee — APPROVED** [review]
LGTM

File: server/views/pages/checkTransfer.njk [hunk 3]
```diff
@@ -1,75 +1,97 @@
 {% extends "../partials/layout.njk" %}
-{% set pageTitle = "Check transfer" %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
```
Comment (andrewrlee) [inline server/views/pages/checkTransfer.njk:2]: meh.. haven't you been adding these to the layout template?

Comment (GurnankCheema) [reply server/views/pages/checkTransfer.njk:2]: yes, I have. 
I'll move it over in next ticket

**Pull request merged**